conductor_api/src/conductor/base.rs

```diff
@@ -629,6 +629,64 @@ impl Conductor {
                     ))
                 })?;
 
+                // This is where we are checking the consistency between DNAs: for now we compare
+                // the hash provided in the TOML Conductor config file with the computed hash of
+                // the loaded dna.
+                {
+                    let dna_hash_from_conductor_config = HashString::from({
+                        match dna_config.hash {
+                            Some(v) => v,
+                            None => {
+                                // Let's warn the user that we encountered an empty DNA hash in the Conductor
+                                // config that is meant to be skipped by the consistency checking mechanisim
+                                context.log("warn/Conductor: Invalid DNA hash encountered in the Conductor configuration.");
+                                String::from("")
+                            }
+                        }
+                    }).to_owned();
+
+                    let dna_hash_computed = &dna.address();
+
+
+                    match Arc::get_mut(&mut self.dna_loader)
+                        .expect("Fail to get a mutable reference to 'dna loader'.")(&dna_file) {
+                        // If the file is correctly loaded, meaning it exists in the file system,
+                        // we can operate on its computed DNA hash
+                        Ok(dna) => {
+                            let dna_hash_computed_from_file = HashString::from(dna.address());
+                            Conductor::check_dna_consistency_from_all_sources(
+                                &context,
+                                &dna_hash_from_conductor_config,
+                                &dna_hash_computed,
+                                &dna_hash_computed_from_file, &dna_file)?;
+                        },
+                        Err(_) => {
+                            // But if it doesn't exist, like in every unit test, we warn the user
```

Comment:
Is this still the case?

Reply:
No it's not, so I rewrote the log message. Thanks!